ch_tools/monrun_checks/ch_orphaned_objects.py

```diff
@@ -52,7 +56,7 @@ def orphaned_objects_command(
         return Result(CRIT, str(e))
 
     total_size = state.orphaned_objects_size
-    error_msg = state.error_msg
+    error_msg = _error_message_format(state.error_msg)
```

Comment:
Consider putting this action [there](https://github.com/yandex/ch-tools/blob/35d396c5a46f143c0168d81eecafef25296aac9a/ch_tools/common/result.py#L46) to make it more general

Reply:
thks, that's much better. done